build-tests/heft-sass-test/src/styles.sass

```diff
@@ -24,5 +24,5 @@ html, body
   width: 400px
 
 .exampleButton
-  @include buttono-block()
-  @include buttono-style-modifier($background-color: mediumorchid)
+  @include buttono.buttono-block()

```

Comment:
Also in a similar vein, the separate test which supports the @use syntax we can include this variant as well.